sdk/core/azure-core-test/inc/azure/core/test/test_base.hpp

```diff
@@ -7,11 +7,20 @@
 
 #include <gtest/gtest.h>
 
+#include <azure/core/credentials/credentials.hpp>
+#include <azure/core/credentials/token_credential_options.hpp>
+#include <azure/core/internal/client_options.hpp>
+#include <azure/core/internal/diagnostics/log.hpp>
+
 #include "azure/core/test/interceptor_manager.hpp"
 #include "azure/core/test/network_models.hpp"
 #include "azure/core/test/test_context_manager.hpp"
 
+#include <chrono>
 #include <memory>
+#include <thread>
+
+using namespace std::chrono_literals;
```

Comment:
Avoid to use `using namespace` in header declarations, so that when the header is included, you have the namespace included. This means that for that default `1s` value in the function below, you're gonna need to use the non-literal version.

Reply:
I wanted to add it here excatly to make all tests cases to use this namespace.

This header is not included by customers, only by our test cases, and I want to have the literals available in all test cases.